src/cook-web/src/app/admin/operations/page.tsx

```
@@ -936,6 +1089,63 @@ const OperationsPage = () => {
+      ]);
+    } catch (error) {
+      if (requestId !== copyRequestIdRef.current) {
+        return;
+      }
+      setCopyError(
+        error instanceof Error ? error.message : t('common.core.unknownError'),
+      );
+      setCopyLoading(false);
+    } finally {
+      if (requestId !== copyRequestIdRef.current) {
+        return;
```

Biome (2.4.15)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)